cmake.md

```diff
@@ -0,0 +1,30 @@
+## CMake build system
+### Terminology
+
+The CMake system in symbiflow-arch-defs uses the following hierarchy:
+
+* ARCH - The architecture of FPGA.  This is used to capture architecture level parameters (e.g. tech libraries).
```

Comment:
Architecture is the high level grouping of a set of "similar" FPGA types. An example would be the "Lattice iCE40 Series" (`ice40`) or the "Xilinx Artix 7 Series" (`artix7`).

Reply:
Fixed